gazebo_ros/test/plugins/sdf_node.cpp

```diff
@@ -39,43 +41,79 @@ void SDFNode::Load(gazebo::physics::WorldPtr, sdf::ElementPtr _sdf)
   auto node = gazebo_ros::Node::Get(_sdf);
   assert(nullptr != node);
 
+  node->declare_parameter("declared_string", rclcpp::ParameterValue("overridden"));
+  node->declare_parameter("declared_int", rclcpp::ParameterValue(123));
+  node->declare_parameter("missing_type", rclcpp::ParameterValue("declared"));
```

Comment:
I think the `rclcpp::ParameterValue(...)` is redundant. `declare_parameter` has a templated version:

https://github.com/ros2/rclcpp/blob/8ff51833ad7bceeb4c4cb2bacfbe0de02da25e5e/rclcpp/include/rclcpp/node.hpp#L422-L428